Description: Emit message when skipping package when audio decoding fail.
Author: Petter Reinholdtsen <pere@hungry.com>
Bug-Debian: https://bugs.debian.org/789254
Forwarded: no
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2015-06-23

diff --git a/libffsox-2/ffsox_frame_reader.c b/libffsox-2/ffsox_frame_reader.c
index d4f9f1c..0bda906 100755
--- a/libffsox-2/ffsox_frame_reader.c
+++ b/libffsox-2/ffsox_frame_reader.c
@@ -146,6 +146,7 @@ static int frame_reader_run(frame_reader_t *n)
 
       if ((size=avcodec_decode_audio4(cc,frame,&got_frame,pkt))<0) {
         // skip the package.
+        DMESSAGE("decoding audio, skipping audio package");
         pkt->size=0;
         return 0;
       }
